simulators/rpc-compat/Dockerfile

```diff
@@ -18,12 +18,14 @@ FROM ubuntu:22.04
 
 RUN apt update && apt install wget -y
 
-# Use these for amd-based devices
-RUN wget http://nz2.archive.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.0g-2ubuntu4_amd64.deb
-RUN dpkg -i libssl1.1_1.1.0g-2ubuntu4_amd64.deb
-# Use these for arm-based devices
-#RUN wget http://ports.ubuntu.com/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2_arm64.deb
-#RUN dpkg -i libssl1.1_1.1.1f-1ubuntu2_arm64.deb
+RUN export ARCH=$(uname -m | sed -e "s:aarch64:arm64:" -e "s:x86_64:amd64:") \
+  && export BASE=$(case $ARCH in ("amd64") echo "nz2.archive" ;; ("arm64") echo "ports" ;; (*) echo "ports" ;; esac) \
+  && wget http://${BASE}.ubuntu.com/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2_${ARCH}.deb \
+  && dpkg -i libssl1.1_1.1.1f-1ubuntu2_${ARCH}.deb
+
+RUN 
+
+RUN sudo apt-get install 
```

Comment:
```suggestion
```

We should probably remove these since they don't do anything?

Reply:
Sorry, I was updating my branch to test on another laptop.